addons/purchase/views/purchase_views.xml

```diff
@@ -534,14 +534,6 @@
             </field>
         </record>
 
-        <record id="purchase_order_action_generic" model="ir.actions.act_window">
```

Comment:
correct but wrong ref in commit message https://github.com/odoo/odoo/commit/0e4f3bb95991261c04c08e72b28430acfabdecd1#diff-b2428a6acab359488e97bb56f33d555aR33

Reply:
2c5022b4f33b1406736d0f9a0bee76942ba0bf7f , yes my bad, this one is correct.